morpheus/_lib/include/morpheus/messages/multi.hpp

```diff
@@ -232,38 +236,69 @@ class DerivedMultiMessage<DerivedT>
     }
 };
 
+/**
+ * @brief  This class holds data for multiple messages at a time. To avoid copying data for slicing operations, it
```

Comment:
One of the more confusing things about Morpheus is that we use "message" to refer to two different things:
1) Instances of a message class
2) Rows in a DataFrame

Maybe add something explicit to indicate we are talking about rows in the dataframe like:
```suggestion
 * @brief  This class holds data for multiple messages (rows in a DataFrame) at a time. To avoid copying data for slicing operations, it
```